### ISSUE TITLE
Not available in UK

### ISSUE THREAD

**Issue by BillTribble** [habitlab/HabitLabMobile#10]
Can you fix this? Can't install from Play UK. Thanks!

**Comment by BillTribble** [comment]
Many thanks Drew I'll try it out!

**Comment by BillTribble** [comment]
Hi Drew - installs fine, but crashes at 1st step after entering name. Using
on Essential PH-1.

**Comment by DrewGregory** [comment]
Yep, before this update, we were having a lot of crash issues. At last, I have published an app update. Hopefully, the app will be more stable. Let me know if you are still running into issues. We also now support signing into your Google account to sync data across devices (your email will be anonymized) and have fixed a lot of bugs. We will soon ship an update to the browser so that you can visualize your usage across different browsers and android devices under the same Google Account. Let me know if you run into any issues.

Thanks again for being so patient!

**Comment by BillTribble** [comment]
Great stuff, seems to work now!

One suggestion: 'groups' for apps. Right now I can choose say 15 mins a day
for Insta and Twitter each, but I'd prefer to be able to say: No more than
20 mins for either Insta or Twitter each day.

Cheers!

**Comment by DrewGregory** [comment]
Hi @BillTribble, great idea for a new feature! I will reopen this issue. Currently, I am making to sure to smooth out any bugs. However, this may be a cool idea to pursue once the app is sufficiently established and stable.

**Comment by BillTribble** [comment]
Awesome, thanks Drew! A few backgrounds crashes here at the moment but
loving the app so far - really interesting.

Will be sharing with my UX students here at General Assembly!

**Comment by DrewGregory** [comment]
That would be fantastic! Thank you so much! Of course, feel free to have them post here with design recommendations or feedback.

**Comment by BillTribble** [comment]
Absolutely, will do!

Sadly, most of em are on iPhones, but I should be able to get one or two
Android owners to try it out :)

Such a shame iOS is so limited for this kind of thing.
